### ISSUE TITLE
Give feedback with spinners

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/notifications#115]
* Dismiss
* Dismiss all
* Action button

**Comment by MorrisJobke** [comment]
Or disable those buttons (that is for example how github does it). cc @nextcloud/designers

**Comment by jancborchardt** [comment]
For the dismiss buttons, we should directly collapse the entry (like we do on Mail message deletion).

When deleting all, also collapse all. And in that case an "Undo" would be good too.

Spinners for feedback are nice additionally, but we really need to immediately show action, even if it's just "faked" and the request still runs in the background. Otherwise the interface feels sluggish.

**Comment by nickvergessen** [comment]
@jancborchardt well, it's easy and quickly done to move the "feedback" from inside the success to the action itself. But then I wonder: should they re-appear on failure?

**Comment by jancborchardt** [comment]
@nickvergessen yes, on failure they can re-appear. :) But it’s key to vanish them as quickly as possible when they are deleted, and in most cases it is probably successful.

**Comment by MorrisJobke** [comment]
I'll unassign the milestone.
